content/blog/posts/student-story-whitehet.rst

```diff
@@ -0,0 +1,41 @@
+Reflections on My Time at the Open Source Lab, Travis Whitehead
+---------------------------------------------------------------
+:author: Travis Whitehead
+:date: 2019-04-24
+:slug: student-story-whitehet
+:tag: student-stories
+ 
+Reflections on My Time at the Open Source Lab
+by Travis Whitehead
+
+Working at the Open Source Lab has been the highlight of my computer science experience at Oregon State University. It
+was just by chance that I came across a job listing for the OSL. I had never heard of the organization, and it certainly
+was not a factor in my decision to pursue computer science at Oregon State University.
+
+I'd been running Linux as my primary operating system since high school, and over time I found myself becoming more and
+more deeply invested in the ideological underpinnings of FOSS (Free Open-Source Software). I appreciated FOSS’
+transparency, and the benefits available through free licenses that allow anyone to use the software, change how it
+works, repurpose it, and distribute it.
+
+Despite my strong interest in free software, I never imagined myself in the position of getting paid to contribute to
+open source. At the OSL I learned valuable skills and gained work experience, but the biggest thing to me was that I was
+able to do work that was ethical and important.
+
+In a world shaped by a for-profit economy, our interaction with software and intellectual property is exclusive. If
+users cannot afford to pay for software, they are excluded access to software or must access it illegally. Or worse, we
+become the products ourselves, subjected to all kinds of data collection and surveillance in exchange for access to
+services. The Open Source Lab offered me the opportunity to support open-source software projects, ultimately allowing
+me to contribute to The Commons, and better the world that we live in. It's been very fulfilling for me to know that the
+projects we support create solutions that anybody and everybody may use.
+
+Ethics aside, the work itself has provided me many opportunities to learn things that I couldn’t in the classroom.
+Experience with configuration management and automation tooling reshaped how I manage my personal systems at home. And I
+worked in in a real datacenter! Tinkering with powerful hardware in a real production environment is way cooler than any
+academic project.
+
+Looking forward, I hope that the Open Source Lab continues to grow and expand so that more students may have these same
+opportunities. The OSL is truly one of a kind, and I feel really thankful to have been able to work with the lab for the
+past several years.
+
+Excitingly, this is my last term at Oregon State. I'll be going on to work with Tag1 Consulting, where I will continue
```

Comment:
Why don't you add a link to Tag1's website here?

Reply:
Added a link.